Assets/Scripts/EOSStoreManager.cs

```diff
@@ -55,10 +55,15 @@ public void OnLoggedIn()
         }
 
 #if !EOS_DISABLE
-        void IEOSOnAuthLogin.OnAuthLogin(LoginCallbackInfo loginCallbackInfo)
+        public void OnAuthLogin(LoginCallbackInfo loginCallbackInfo)
         {
             OnLoggedIn();
         }
+
+        public void OnAuthLogout(LogoutCallbackInfo logoutCallbackInfo)
+        {
+            OnLoggedOut();
+        }
 #endif
```

Comment:
Before merging, make certain that if you define `EOS_DISABLE` in the scripting defines section of the player build settings, this does not break (I'm pretty sure that it will, and some additional `#if` statements will need to be added).

Reply:
I have tested this and determined that it is not an issue, since when `EOS_DISABLE` is defined, everything within `Assets/Scripts/` is excluded from the build (including `EOSStoreManager.cs`).